guides/v2.2/rest/bulk-endpoints.md

```diff
@@ -11,7 +11,9 @@ functional_areas:
 Bulk API endpoints differ from other REST endpoints in that they combine multiple calls of the same type into an array and execute them as a single request. The endpoint handler splits the array into individual entities and writes them as separate messages to the message queue.
 
 {:.bs-callout .bs-callout-tip}
-Use the `bin/magento queue:consumers:start async.operations.all` command to enable bulk endpoint processing.
+Use the `bin/magento queue:consumers:start async.operations.all` command to enable bulk endpoint processing, after configure the message queues. For more details see: [
```

Comment:
Basically, the [mentioned](https://devdocs.magento.com/guides/v2.3/config-guide/mq/manage-message-queues.html) article describes how to configure message queues using CRON consumers runner which is not default MQ processor in Magento. By default, Magento uses RabbitMQ (AMQP) for processing message queues and its configuration process is slightly different. So, I would mention the [following](https://devdocs.magento.com/guides/v2.3/install-gde/prereq/install-rabbitmq.html) article first of all and then the [alternative]((https://devdocs.magento.com/guides/v2.3/config-guide/mq/manage-message-queues.html)) way of configuring the message queues. 

Thank you!

Reply:
I don't agree with this comment, Magento both uses MysqlMQ and RabbitMQ out of the box, it depends on which queue and how it's defined in the xml configuration files for that particular queue.
Cron's don't really have anything to do with this, they are used to spawn message queue consumers, doesn't matter for which broker (RabbitMQ or MysqlMQ).

As for bulk operations, I think Magento defaults to the RabbitMQ at the moment, so setting up RabbitMQ is probably a requirement before bulk operations can work.